src/app/_schemas/blogPostDataSchema.ts

```diff
@@ -0,0 +1,23 @@
+import { z } from 'zod'
+
+import { DynamicBaseDataSchema } from '@/schemas/dynamicDataBaseSchema'
+
+import {
+  getCategorySettings,
+  createCategorySchema,
+} from '@/utils/categoryUtils'
+
+const { validCategoryOptions } = getCategorySettings('blog_posts')
+
+const categorySchema = createCategorySchema(validCategoryOptions)
+
+export const BlogPostDataSchema = DynamicBaseDataSchema.extend({
+  slug: z.string(),
+  title: z.string(),
+  category: categorySchema,
+  description: z.string(),
+  recommendedPosts: z.array(z.string()).optional(),
```

Comment:
Yes! It can be an array of blog post slugs. But since we aren't using this feature at the moment, I'm going to remove it all together so it's less confusing. Thank you.